eo-maven-plugin/README.md

```diff
@@ -32,6 +36,14 @@ create a file `pom.xml` with this content (it's just a sample):
   [...]
   <build>
     <plugins>
+      <plugin>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <version>3.8.1</version>
+        <configuration>
+          <source>8</source>
+          <target>8</target>
+        </configuration>
```

Comment:
@ivan-egorov42 Do we really need this configuration?

Reply:
@Graur we have error without it
```
[ERROR] Source option 5 is no longer supported. Use 6 or later.
[ERROR] Target option 1.5 is no longer supported. Use 1.6 or later.
```

Reply:
@ivan-egorov42 Indeed, let's keep it